pkg/datapath/linux/config/config.go

```diff
@@ -895,7 +878,7 @@ func vlanFilterMacros(nativeDevices []*tables.Device) (string, error) {
 	vlansCount := 0
 	for _, v := range vlansByIfIndex {
 		vlansCount += len(v)
-		slices.Sort(v) // sort Vlanids in-place since netlink.LinkList() may return them in any order
+		slices.Sort(v) // sort Vlanids in-place since safenetlink.LinkList() may return them in any order
```

Comment:
Note to reviewers: This is necessary so `check-safenetlink.sh` doesn't complain about it.